components/filter/test/filter.test.js

```diff
@@ -772,7 +772,7 @@ describe('d2l-filter', () => {
 			});
 			expect(elem._dimensions[1].values[0].text).to.equal('Test');
 			expect(elem._dimensions[1].values[0].selected).to.be.true;
-			expect(updateStub).to.be.calledOnce;
+			expect(updateStub).to.be.called;
```

Comment:
This calledOnce test was failing since requestUpdate is now being called by ReactiveElement (new library inside lit 2). Example call: 
`requestUpdate('_totalAppliedCount', 2, { type: [Function: Number], attribute: false })  at HTMLElement.set [as _totalAppliedCount] (http://localhost:8001/node_modules/@lit/reactive-element/reactive-element.js:6:1292)
 `

Sinon can not tell the difference between the calls since they have the same name. Sinon stubs CAN spy on calls with arguments, but there is currently no way to check how many times requestUpdate was called with NO args (since the stub matches calls with arbitrary amounts of arguments).

Reply:
Yeah, looks like Lit is now also calling `this.requestUpdate` whenever it detects a property has changed, so it gets changed multiple times as I update the `_totalAppliedCount`.  I'm good with these changes - Lit _should_ be smart and properly deduping and only _actually_ updating once, so we don't really need to test that.